### ISSUE TITLE
Failed to start a new Google account.

### ISSUE THREAD

**Issue by ValorRed** [Necrobot-Private/NecroBot#129]
In 0.9.9.2, created a new gmail account, pass the 2 tier verification, setup the recovery email. Still getting this:

[22:26:56] Logging in using Google
[22:26:58] User credentials are invalid and login failed.

**Comment by ajns73** [comment]
This bot will not work with 2-step verification to my knowledge.

**Comment by ValorRed** [comment]
I think I had it worked before. Probably 0.9.8.5, when I applied a new Google account. And maybe another time before that.
Whenever there was a major concern of ban, I start using a new version of bot with a new account.

What do you mean by "app specific password"? My    "AuthType": "google", not "ptc"

**Comment by mjusuf** [comment]
@ValorRed generate your app password here: https://security.google.com/settings/security/apppasswords

**Comment by ajns73** [comment]
exactly. now it is a single challenge to login instead of double

**Comment by ValorRed** [comment]
Sorry, by 2-tier, I meant when I created the new account, google asked me to provide a phone# to text me a code/or voice msg a code. I have to enter the code, once and only once, before I start using gmail.
I did it successfully.  
All these has been done using PC. I have not use this account touch a mobile device at all. Only having 1 Android device.

Tried that link above see this:
![image](https://cloud.githubusercontent.com/assets/20829876/18498527/0b479286-7a05-11e6-9bff-1874563dc64a.png)

**Comment by ajns73** [comment]
@ValorRed Ahhhh, right, you would see that because you have not enabled 2-step verification. If 2-step verification is not enabled and you are still getting errors, it's weird.

**Comment by ValorRed** [comment]
Yes. Still the error:
![image](https://cloud.githubusercontent.com/assets/20829876/18498588/aa241172-7a05-11e6-9e07-9f26800f3f86.png)

**Comment by CreativiTimothy** [comment]
This worked for me. Log out of all your browsers on Google (all your devices if you can). Then, log back into your account on Google Chrome of whatever browser. Finally, run Necrobot. It should work. IF it doesn't work, try signing out again and signing back in.

**Comment by LobsterX** [comment]
when i create google accounts, i dun get the prompt to verify via phone or mobile.
this happens[the verification] only when using a browser to create an account.

i bypass this by simply using the add google account in my settings to create a new email. and skip entering a phone number.

**Comment by ValorRed** [comment]
Tried log out all accounts in Chrome, Close chrome and openagain, relogin with the new account, 
Still not working. Please help. Any log I can provide to help to track the issue?

**Comment by Silph-Road** [comment]
@ValorRed 
Closing this issue, its a duplicate of :
https://github.com/Necrobot-Private/NecroBot/issues/143
